### ISSUE TITLE
Should DACS promote structured dates more?

### ISSUE THREAD

**Issue by gwiedeman** [saa-ts-dacs/dacs#53]
## Link to relevant DACS principle

e.g. [Principle 8. Archival description should be easy to use, re-use, and share.](https://saa-ts-dacs.github.io/dacs/04_statement_of_principles.html#8-archival-description-should-be-easy-to-use-re-use-and-share)

## Describe how DACS does not currently meet this principle

Well-structured dates are important for aggregation and re-usability. Currently DACS provides lots of useful guidance for human-readable dates, but does not even suggest including a structured or machine-focused representation for dates, as is common practice. Is this out-of-scope and left to data structured standards, or it is useful for DACS to recommend including well-structured dates as additional representation for human readable dates. For example, a description might contain a date of `2020 March 8`, as well as a second representation of `2020-03-08`. Perhaps some common normalization practices for circa dates and undated could then emerge. A requirement to makes some kind of ISO 8601 compliant representation, may be too extreme, but its weird that DACS does not encourage this, as its an established best practice.

## I expect that this will require a

- [ ] Minor change to DACS
- [x] Major change to DACS

**Comment by regineheberlein** [comment]
Yes! Being implementation-neutral should not entail not being implementation-ready, especially in an environment that has made a significant shift, since DACS was first written, towards software being among the prime consumers of our descriptive efforts.
